src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -857,6 +849,34 @@ public virtual HtmlString SetAntiforgeryCookieAndHeader()
             return HtmlString.Empty;
         }
 
+        public static bool ShouldRenderAttributeValue(object rawAttributeValue)
```

Comment:
`static` methods normally come before instance methods. At least move these two above `WriteUnprefixedAttributeValueTo()` to match the order the three are used.